template.yaml

```diff
@@ -61,12 +89,12 @@ Resources:
                 Effect: Allow
 
 Outputs:
-  # ServerlessRestApi is an implicit API created out of Events key under Serverless::Function
-  # Find out more about other implicit resources you can reference within SAM
-  # https://github.com/awslabs/serverless-application-model/blob/master/docs/internals/generated_resources.rst#api
-  ProgramCodeRulesApi:
+  ApiOrigin:
     Description: "API Gateway endpoint URL for Prod stage for function"
-    Value: !Sub "https://${ServerlessRestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/program-codes/"
+    Value: !Sub "https://${RestApi}.execute-api.${AWS::Region}.amazonaws.com/Prod/"
+  ProgramCodeRulesEndpoint:
+    Description: "Custom endpoint URL for Prod stage for function"
```

Comment:
now that you multiple paths, consider updating the description to be more specific?

no output for the owner email endpoint?

Reply:
whoops, updating